bicycleparameters/main.py

```diff
@@ -877,7 +877,7 @@ def plot_eigenvalues_vs_speed(self, speeds, fig=None, generic=False,
         figsize = [figwidth, figwidth * goldenMean]
         params = {#'backend': 'ps',
             'axes.labelsize': 8,
-            'text.fontsize': 10,
+            #'text.fontsize': 10, # removed because out of date
```

Comment:
What are the consequences of this? Are the fontsizes now too large?

Reply:
We debugged that line a little while back. It's a method that isn't available for the matplotlib figures anymore and it was preventing plots from being drawn properly.

Reply:
I remember that, but what is the effect of removing this?

Reply:
It makes it impossible to draw plots. The error is this;

KeyError: 'text.fontsize is not a valid rc parameter (see rcParams.keys() for a list of valid parameters)'

Reply:
Sorry, I wasn't clear. I'd like to know if this is removed, do the font sizes change to values that change the plot outputs? If so, then you need to implement an alternative to set these font sizes correctly.

Reply:
I do not know. With the line commented, the plot looks fine and I don't think I would want to change the font sizes anyways. When I try to run it without commenting out that line, it immediately fails and does not draw the plot at all. I can't assess the change. This is what I ran;
```
import bicycleparameters as bp
import numpy as np
import os

speed = np.linspace(0,10,num=100)
bike = bp.Bicycle('Benchmark', pathToData=os.getcwd()+'\\data')
bike.plot_eigenvalues_vs_speed(speed, show=True)
```

This is what I got
```
  File "C:\Users\xyzasdf\AppData\Local\Programs\Python\Python38\lib\site-packages\matplotlib\__init__.py", line 811, in __setitem__
    raise KeyError(
KeyError: 'text.fontsize is not a valid rc parameter (see rcParams.keys() for a list of valid parameters)'
```